src/schedule/schedule.11tydata.js

```diff
@@ -0,0 +1,45 @@
+const { google, outlook, office365, yahoo, ics } = require('calendar-link');
+const simpleIcons = require('simple-icons');
+const { isBefore } = require('date-fns');
+
+module.exports = {
+	eleventyComputed: {
```

Comment:
I put my thoughts in the **#frameworks** channel, but I'll duplicate them here for the sake of colocation and rediscoverability:

> :rotating_light: **Personal philosophy alert!** :rotating_light:
>
> I think computed data is a great way to approach this. However, I don't think we should return _markup_ from computed data. I personally would prefer to see markup in the layouts directly, with its fields and attributes populated by computed data. So `href`s would be computed data; the anchor tags are in the layouts. That's my two cents.
>
> So, I'd use eleventyComputed to create a `googleCalendar` href property, an `outlookCalendar` href property, and so forth. Then I'd use those properties in my layouts, with the handlebars.
>
> I'd personally also pull out the different calendars' SVGs as partials in `_includes`.
>
> So one link in `event.html` might look like:
>
> ```liquid
> <a
>   href="{{ googleCalendar }}"
>   aria-label="Add to Google Calendar"
> >
>  {% include "partials/google-calendar.svg" %}
> </a>
> ```

Reply:
i agree. mostly just wanted to keep this branch-to-branch PR as minimal a changes as possible.
using the templates for `svg` might require a reward of how the svgs are imported and displayed.

would you like to resolve that here or on @roman's branch (after this change is merged fix)?